cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/CgmesTerminal.java

```diff
@@ -43,6 +44,7 @@ public CgmesTerminal(PropertyBag t) {
         // (TODO(Luma): missing reference here)
 
         this.connected = t.asBoolean("connected", false);
+        this.inService = t.containsKey("inService") ? Boolean.parseBoolean(t.getId("inService")) : null;
```

Comment:
In CGMES the `inService` attribute comes from the `SvStatus` object, that is linked to the `ConductingEquipment`, not the `Terminal`. I know the SPARQL for terminals returns the `inService` property of its `ConductingEquipment`, and probably this is the best way of handling it, save it at each terminal. We just have to keep in mind that this inService refers to the status of the equipment, and should be updated when topology is calculated. For branches, if one terminal is disconnected, it does not mean that the equipment is out of service.

From CGMES documentation: _A valid case is that conducting equipment can be connected in one end and open in the other. In particular for an AC line segment, where the reactive line charging can be significant, this is a relevant case._